enderio-machines/src/main/java/com/enderio/machines/common/blockentity/SlicerBlockEntity.java

```diff
@@ -0,0 +1,104 @@
+package com.enderio.machines.common.blockentity;
+
+import com.enderio.machines.common.MachineTier;
+import com.enderio.machines.common.blockentity.base.PoweredCraftingMachine;
+import com.enderio.machines.common.blockentity.task.PoweredCraftingTask;
+import com.enderio.machines.common.init.MachineCapacitorKeys;
+import com.enderio.machines.common.init.MachineRecipes;
+import com.enderio.machines.common.io.item.MachineInventory;
+import com.enderio.machines.common.io.item.MachineInventoryLayout;
+import com.enderio.machines.common.menu.SlicerMenu;
+import com.enderio.machines.common.recipe.SlicingRecipe;
+import net.minecraft.core.BlockPos;
+import net.minecraft.world.Container;
+import net.minecraft.world.entity.player.Inventory;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.inventory.AbstractContainerMenu;
+import net.minecraft.world.item.AxeItem;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.ShearsItem;
+import net.minecraft.world.item.Tiers;
+import net.minecraft.world.level.block.entity.BlockEntityType;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraftforge.common.TierSortingRegistry;
+import net.minecraftforge.items.wrapper.RecipeWrapper;
+import org.jetbrains.annotations.Nullable;
+
+public class SlicerBlockEntity extends PoweredCraftingMachine<SlicingRecipe, Container> {
```

Comment:
Slicer or SliceAndSlice or SliceNSplice?

Reply:
I think SliceAndSplice is most readable, and its fits better than Slicer. Going with that.